wherehows-web/app/components/datasets/owners/suggested-owner-card.ts

```diff
@@ -0,0 +1,18 @@
+import DatasetAuthorComponent from 'wherehows-web/components/dataset-author';
+
+export default class DatasetsOwnersSuggestedOwnerCard extends DatasetAuthorComponent {
+  /**
+   * Sets the html tag binded to the element generated by this component
+   */
+  tagName = 'div';
+
+  /**
+   * Sets the class names binded to the html element generated by this component
+   */
+  classNames = ['dataset-authors-suggested__card'];
+
+  /**
+   * Sets the class names that are triggered by certain properties on the component evaluating truthy
+   */
+  classNameBindings = [];
```

Comment:
Can we add a comment here to make clear, that this is intended to override the parent attribute? My first thought reading this was, there's no point. Same comment for the `tagName` property as well. Thanks

Reply:
Good point, will do.